src/index.html

```diff
@@ -1,13 +1,73 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="style.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock
+            (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
```

Comment:
use semantic tags for text content
```suggestion
          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock
            (MNED2UA/A)
          </h2>
          <p class="card__code">Product code: 195434</p>
```